onnxruntime/core/flatbuffers/flatbuffers_utils.cc

```diff
@@ -18,14 +18,24 @@ namespace experimental {
 namespace utils {
 
 #if !defined(ORT_MINIMAL_BUILD)
+
+flatbuffers::Offset<flatbuffers::String> SaveStringToOrtFormat(flatbuffers::FlatBufferBuilder& builder,
+                                                               bool has_string, const std::string& src) {
+  if (has_string)
+    return builder.CreateString(src);
+
```

Comment:
Can we infer has_string from src? Don't think we need to differentiate between 'not set' and 'set to empty string' for these fields, so shouldn't matter if serializing an empty string deserializes as a 'not set' value.

Reply:
The empty string and not set is different in the serialized output of protobuf, not sure in reality how many strings are actually set to "" instead of not set, since we use serialized protobuf to compare values in test now, would keep it this way

---
In reply to: [497992150](https://github.com/microsoft/onnxruntime/pull/5331#discussion_r497992150) [](ancestors = 497992150)